src/cli.ts

```diff
@@ -52,9 +54,15 @@ interactive({
   .then((result) => {
     result.pattern = result.pattern || '**';
     clipboardy.writeSync(result.pattern);
-    console.log(`Pattern: ` + chalk.cyan(result.pattern));
-    console.log(`Matched: ` + result.files.length);
-    console.log(chalk.yellow.dim`Copied pattern to clipboard!`);
+    console.error(`Pattern: ` + chalk.cyan(result.pattern));
+    console.error(`Matched: ` + result.files.length);
+    console.error(chalk.yellow.dim`Copied pattern to clipboard!`);
```

Comment:
Using `console.error` will write to `stderr` which is an unfortunate name, but it is file descriptor 2, and not the "default" one. So `globitor | xargs rm` for instance will still print `Pattern: ... Matched: ...` to the screen, and will remove the files. 😃